mxcube3/ui/components/Main.jsx

```diff
@@ -1,30 +1,21 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import SampleQueueContainer from '../containers/SampleQueueContainer'
 import MethodContainer from '../containers/MethodContainer'
-import NavBar from './NavBar'
+import Login from '../containers/LoginContainer'
 import { ErrorNotificationPanel } from './Logging'
 import PleaseWaitDialog from './PleaseWaitDialog'
 import './Main.css'
 
 export default class Main extends React.Component {
     render() {
-        return (<div className="container-fluid"> 
+        return ( <div align="center">
```

Comment:
I thought we could have weird layout problems if the main div in a Bootstrap based app is not a container class,
due to the margins or other properties of other classes like `row` for example.